sof_wrapper/api/fhir.py

```diff
@@ -19,6 +19,42 @@ def collate_results(*result_sets):
     return results
 
 
+def add_cds_extensions(med):
+    """Add FHIR extensions as necessary to support frontend CDS"""
+
+    expected_supply_duration = med.get('dispenseRequest', {}).get('expectedSupplyDuration', {}).get('value')
+    quantity = med.get('dispenseRequest', {}).get('quantity', {}).get('value')
```

Comment:
@ivan-c Ivan, will `dispenseRequest.quantity` or `expected_supply_duration` only be present in a PMP med? Just checking...  Look like you are looking for the presence of one of those before addint the doseInstruction below.

Reply:
> will `dispenseRequest.quantity` or `expected_supply_duration` only be present in a PMP med

Not necessarily, they could be present in EHR meds too.

It's not certain what fields an EHR will have filled out, so I wanted to attempt to "polyfill" whatever data (whether it's from the PDMP or EHR) we get, so the CQL can operate on it.